pkg/loop/internal/core/services/keyvaluestore/keyvalue_store_test.go

```diff
@@ -0,0 +1,61 @@
+package keyvaluestore
+
+import (
+	"context"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/emptypb"
+	"testing"
```

Comment:
Is the linter complaining about the unsorted imports? Typically we sort/group by std lib, third parties, smartcontractkit, then local repo/module. Configuring your IDE to use goimports with `-local github.com/smartcontractkit` does most of the work for you.

Reply:
It looks to be passing the lint (though perhaps I should still update goimports?)  the failing build step looks like it is because I need to push the corresponding changes on the chainlink repo (?) - I understand I just need to ensure that both PRs are ready to go to minimise impact and then merge both as close together as possible, there is some issue on the chainlink PR build -  I'll look to unblock that shortly.

Reply:
> It looks to be passing the lint (though perhaps I should still update goimports?) 

Still worth updating. Maybe we have test files omitted from the linter, but we should update that. Many rules are always important.